README.rst

```diff
@@ -102,7 +106,12 @@ using:
 
    user@host:~$ gpg --export <long_key_ID> > <long_key_ID>
 
-And then specify key ID to person mapping in the ``keyids`` file.
+And then specify the key ID to person mapping in the ``keyids`` file.
+
+Note that you should be reasonably confident that `no
+one has ever had a copy of your private key
+<https://www.wired.com/2014/10/snowdens-first-emails-to-poitras/>`_.
```

Comment:
The linked article does not contain `gpg` or `gnupg` strings at all. Perhaps users should be directed to a better source of valid information, such as [OpenPGP Best Practices](https://help.riseup.net/en/security/message-security/openpgp/best-practices)?